README.md

```diff
@@ -2852,6 +2853,38 @@ Server: spray-can/1.3.3
 }
 ```
 
+## PATCH /api/workflows/:version/labels
+
+This endpoint is used to update multiple custom labels for an existing workflow. When supplying a custom label with a label key unique to the workflow submission, a new custom label key/value entry is added to the workflow metadata. 
+When supplying a custom label with a label-key that is already associated to the workflow submission, the original label value is updated with the newly supplied custom label.
```

Comment:
I think I'd just call these "labels" to avoid confusion. IMO having `labels` and `default labels` isn't a big deal and colloquially people are going to say `labels` anyways.